test/desktop-lite/test.sh

```diff
@@ -5,9 +5,29 @@ set -e
 # Optional: Import test library
 source dev-container-features-test-lib
 
+echoStderr()
+{
+    echo "$@" 1>&2
+}
+
+checkOSPackage() {
+    LABEL=$1
+    PACKAGE_NAME=$2
+    echo -e "\n🧪 Testing $LABEL"
+    # Check if the package exists and retrieve its exact version
+    if [ "$(dpkg-query -W -f='${Status}' "$PACKAGE_NAME" 2>/dev/null | grep -c "ok installed")" -eq 1 ]; then
+        echo "✅  Package '$PACKAGE_NAME' is installed."
+    else
+        echo "❌ Package '$PACKAGE_NAME' is not installed."
+    fi
+}
+
 check "desktop-init-exists" bash -c "ls /usr/local/share/desktop-init.sh"
 check "log-exists" bash -c "ls /tmp/container-init.log"
 check "fluxbox-exists" bash -c "ls -la ~/.fluxbox"
 
+checkOSPackage "if libasound2 exists !" "libasound2"
+checkOSPackage "if libasound2-dev exists !" "libasound2-dev"
```

Comment:
how are the tests passing for noble as well as others distros (eg. jsammy) ? 
Shouldn't the check for `libasound2` fail for noble? I don't see OS related condition to make it happy

Reply:
thanks, done 👍